java/connectors/semantickernel-connectors-memory-jdbc/src/main/java/com/microsoft/semantickernel/connectors/memory/jdbc/JDBCConnector.java

```diff
@@ -81,8 +83,8 @@ public Mono<Void> createTableAsync() {
                                     "CREATE TABLE IF NOT EXISTS "
                                             + TABLE_NAME
                                             + " ("
-                                            + "collection TEXT, "
-                                            + "key TEXT, "
+                                            + "collection TEXT NOT NULL, "
+                                            + "key TEXT NOT NULL, "
                                             + "metadata TEXT, "
                                             + "embedding TEXT, "
                                             + "timestamp TEXT, "
```

Comment:
Is this a TEXT or some other Data type?

Reply:
Solving `timestamp` type in a further PR.